### ISSUE TITLE
Gold nav bar buttons in simple theme don't have proper color contrast

### ISSUE THREAD

**Issue by kevincrafts** [CuBoulder/express#1645]
## Context

When gold buttons are placed on the navigation bar the button text color is white. White & CU Gold do not have sufficient color contrast and black should be used instead.

![screen shot 2017-10-30 at 9 57 34 am](https://user-images.githubusercontent.com/349713/32181009-0a185f9c-bd59-11e7-8bf6-adffef12e1c5.jpg)
